docs/modules/deployment/pages/backup/introduction.adoc

```diff
@@ -0,0 +1,35 @@
+[[backup]]
+= Back up OpenNMS {page-component-title}
+
+This section describes a basic backup process for a default OpenNMS {page-component-title} xref:deployment:core/getting-started.adoc#getting-started-core[installation].
+OpenNMS {page-component-title} binaries, logs, RRD archives, configurations and the database will be backed up.
+
+NOTE: In this example the backup storage path is `/backup` for demonstration purposes.
+
+[[backup-objectives]]
+== Objectives
+
+* Create backup of OpenNMS {page-component-title} components
+* Create PostgreSQL database backup
+* Able to run a restore without having OpenNMS installation packages available
+
+[[backup-requirements]]
+== Requirements
+
+* Linux physical server or a virtual machine running a supported xref:deployment:core/system-requirements.adoc#operating-systems-core[Linux operating system]
```

Comment:
As we are backing up an existing server, this line feels a little redundant

Reply:
Yes. Let's remove it. Thanks!

Reply:
Fixed with https://github.com/OpenNMS/opennms/pull/4484/commits/a19e20a27831c146c9b7355bb8f9c801fc2914e3